iceoryx_utils/include/iceoryx_utils/internal/cxx/vector.inl

```diff
@@ -293,18 +293,24 @@ inline typename vector<T, Capacity>::const_iterator vector<T, Capacity>::end() c
 template <typename T, uint64_t Capacity>
 inline typename vector<T, Capacity>::iterator vector<T, Capacity>::erase(iterator position)
 {
+    iterator ret = nullptr;
+
     if (begin() <= position && position < end())
     {
         uint64_t index = static_cast<uint64_t>(position - begin()) % (sizeof(element_t) * Capacity);
         size_t n = index;
+
+        at(n).~T();
         for (; n + 1u < size(); ++n)
         {
-            at(n) = std::move(at(n + 1u));
+            new (&at(n)) T(std::move(at(n + 1u)));
+            at(n + 1u).~T();
```

Comment:
Could you use upper case literals here as you did in the other files?! You've touched these lines anyway.

Reply:
Was about doing it, but thought it looks awkward when it differs from all functions surrounding it (format-styles mixed each file).... 
Still keen on upper case literals then? Applying it to the whole file?

Reply:
Anticipate a 'change file to use upper case' here... 😄